Symfony/CS/Config/Config.php

```diff
@@ -22,6 +22,8 @@
  */
 class Config implements ConfigInterface
 {
+    const DEFAULT_CACHE_FILE = '.php_cs.cache';
```

Comment:
actually not need for that const. Just initialize `$cacheFile` with the value itself